_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,128 @@
+When you use the `cockroach start-single-node` command to start a single-node cluster with Docker, additional features are available to help with testing and development.
+
+{{site.data.alerts.callout_danger}}
+Single-node clusters are not highly-available or fault-tolerant. They are not appropriate for production use.
+{{site.data.alerts.end}}
+
+- You can optionally set the following [Docker environment variables](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) to create a database and user automatically and to set a password for the user.
+  - `COCKROACH_DATABASE`
+  - `COCKROACH_USER`
+  - `COCKROACH_PASSWORD`
+
+  To prevent data loss, the environment variables are ignored if the `/cockroach/cockroach-data` directory within the container is not empty. To re-initialize the default database, user, and password from the environment variable values, delete the contents of `/cockroach/cockroach-data` within the running container.
+
+- You can optionally mount a directory of initialization scripts into the `docker-entrypoint-initdb.d` directory within the container. These scripts are run after CockroachDB starts and after the database and user (if specified) have been created. The scripts run in the alphanumeric sort order imposed by your locale.
```

Comment:
nit: and after the database and user (if specified via the environment variables)